main.tf

```diff
@@ -1,25 +1,34 @@
 terraform {
   required_providers {
     azurerm = {
-      source = "hashicorp/azurerm"
+      source  = "hashicorp/azurerm"
```

Comment:
Consider moving the provider configuration to a separate `provider.tf` file as per the task requirements.